lxml-stubs/etree.pyi

```diff
@@ -577,7 +577,12 @@ def tostring(
     inclusive_ns_prefixes: Any = ...,
 ) -> _AnyStr: ...
 
-class _ErrorLog: ...
+class _LogEntry:
+    path: Optional[str]
```

Comment:
Adding one attribute will lead to errors about all other attributes. Could you please add the complete list of attributes? There are also several properties.
https://github.com/lxml/lxml/blob/97e6a1a0b4fa39662fd6fd4b49a348619cde165a/src/lxml/xmlerror.pxi#L52-L56